echopype/mask/api.py

```diff
@@ -389,6 +419,9 @@ def apply_mask(
         _variable_prov_attrs(output_ds[var_name], mask)
     )
 
+    # Transpose to original dimension order
```

Comment:
I feel we should not use "transpose" in the comment here since it is a misnomer. How about "Use the original dimension order" ?

Reply:
Yeah I agree, I'll add that